terra-app-helm/aou-rstudio-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,138 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "app.fullname" . }}
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
```

Comment:
I guess some of this is copy-pasted.. but as this is a single-user RStudio deployment I'm guessing we'd always want `replicaCount=1` and not enable HPA?